lib/modules/job/providers/jobs_dashboard.provider.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/cupertino.dart';
+import 'package:project/config/inject.config.dart';
+import 'package:project/core/app_provider.dart';
+import 'package:project/core/exceptions/base.exception.dart';
+import 'package:project/modules/job/models/job.model.dart';
+import 'package:project/modules/job/services/job.service.dart';
+
+class JobsDashboardProvider extends AppProvider {
+  final _jobService = inject.get<JobService>();
+
+  List<Job> _jobs;
+  bool _loading = false;
+  dynamic _error;
```

Comment:
[OPTIONAL] as merge pe nume de genul:
_jobs
_fetchLoading
_fetchJobsError
si fetchJobs(...)

Practic structura asta de 3 field-uri si 1 functie reprezinta un anumit request pe care il poti face in aplicatie, deci ajuta sa aiba nume similare

Reply:
Right, are sens daca provideru asta ajunge sa se ocupe de mai multe lucruri. Gotta get back to this.